pkg/asset/installconfig/gcp/validation.go

```diff
@@ -44,30 +44,20 @@ func Validate(client API, ic *types.InstallConfig) error {
 }
 
 // ValidateInstanceType ensures the instance type has sufficient Vcpu and Memory.
-func ValidateInstanceType(client API, ic *types.InstallConfig, fieldPath *field.Path, instanceType string, req resourceRequirements) field.ErrorList {
+func ValidateInstanceType(client API, fieldPath *field.Path, project, zone, instanceType string, req resourceRequirements) field.ErrorList {
 	allErrs := field.ErrorList{}
 
-	if instanceType == "" {
-		return nil
-	}
-
-	filter := fmt.Sprintf("name = %s", instanceType)
-	instanceTypes, err := client.GetMachineTypes(context.TODO(), ic.GCP.ProjectID, filter)
+	typeMeta, err := client.GetMachineType(context.TODO(), project, zone, instanceType)
 	if err != nil {
-		return append(allErrs, field.InternalError(fieldPath, err))
+		return append(allErrs, field.InternalError(fieldPath.Child("type"), err))
```

Comment:
If the error here is a not-found error, then the field error should be Invalid instead of InternalError.

Reply:
Fixed: https://github.com/openshift/installer/commit/60a57050d1738c13289e883ac5f32cf338d2ea0f